dataactbroker/helpers/validation_helper.py

```diff
@@ -80,6 +81,41 @@ def clean_col(value, clean_quotes=True):
     return value
 
 
+def clean_frame_vectorized(frame: pd.DataFrame, convert_to_str=False, clean_quotes=True):
+    """ In-place strip of surrounding whitespace, make None if empty, and remove surrounding quotes.
+        Note a side-effect of stripping quotes/whitespace is that any non-null, non-string types provided (like int
+        or float) will be converted to their string format in the final result.
+
+        Args:
+            frame: pd.DataFrame to clean
+            convert_to_str: whether to apply .astype(str) on all Series in the DataFrame
+                - If the DataFrame was created or read in from CSV with dtype=str already, this may be unnecessary
+                  and will save on performance if avoided
+                - If left as False and there are mixed types, such as int or float, they will be set to None rather
+                  than converted to their str form
+            clean_quotes: whether to clean extra quotes or not
+
+        Returns:
+            The cleaned DataFrame, with a side-effect of an
```

Comment:
"The cleaned DataFrame, with a side-effect of an"

What.... What is the side-effect!?!??!

Reply:
ha... good catch. Updated docstrings.